### ISSUE TITLE
nPower combo points, old style changed?

### ISSUE THREAD

**Issue by strickland** [lithammer/NeavUI#290]
I don't know but I feel that the old nPower combo's used to show from left to right, or am I just crazy here? I got this vague memory that it used to move from left to right which make the eyes easier to distinguish the points combo's I got, now it is always centered? please someone refresh my memory if I'm delusional here, or my memory is correct.

**Comment by lithammer** [comment]
Yeah I think you're right. Don't remember why or when it changed. I will look into when/if I have some timer over. I actually prefer using [NugComboBar](https://www.wowinterface.com/downloads/info6308-NugComboBar.html) most of the time though.
